eng/Subsets.props

```diff
@@ -369,6 +369,10 @@
       Test="true" Category="clr" Condition="'$(DotNetBuildSourceOnly)' != 'true' and '$(NativeAotSupported)' == 'true'"/>
   </ItemGroup>
 
+  <ItemGroup Condition="$(_subset.Contains('+tools.cdacreadertests+'))">
```

Comment:
You also need to declare the subset above with a `SubsetName` item (that will fix the validation error and also provide some basic docs for the subset)